### PR TITLE
Model/bug fixing video search

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 1]
```diff
@@ -25,31 +25,31 @@
...
+        searchColorSpace = null;
         extractMetadata = Runtime.getRuntime().exec(path);
         readAll();
     }
 
     /**
      * This method read all metadata.
-     *
-     * @return
+     * @return void
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:40]: Void methods don't have @return comments

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 2]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
 
     /**
      * This method read mime type.
-     *
-     * @return
+     * @return void
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:63]: Void methods don't have @return comments

Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:63]: Use descriptive method names

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 3]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
+                searchMimeType = mimeType;
+            } else {
+                searchMimeType = "All";
             }
         }
     }
 
     /**
      * This method returns metadata image height.
+     * @return void
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:82]: Void methods don't have @return comments. Use descriptive names, please

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 4]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
-     */
-    public static String getHeight() {
-        return searchHeight;
-    }
-
     /**
      * This method returns metadata image Width.
+     * @return void
      */
     public static void width(String width) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:99]: Void methods don't have @return comments. Use descriptive names, please

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 5]
```diff
@@ -101,47 +103,74 @@ public static void width(String width) {
...
-    /**
-     * This method returns width image.
-     */
-    public static String getWidth() {
-        return searchWidth;
-    }
-
     /**
      * This method returns metadata image color space.
+     * @return void
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:113]: Void methods don't have @return comments

File: src/main/java/com/fundation/logic/common/MetadataVideoExtractor.java [hunk 6]
```diff
@@ -66,16 +67,19 @@ public void readAll() {
     }
 
     /**
-     * This method get all metadata and set in each method for get a specific metadata.
+     * This method read mime type.
      * @return void
-     * @param searchMimeType
      */
     public static void mimeType(String mimeType) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataVideoExtractor.java:71]: Void methods don't have @return comments. Use descriptive names, please

File: src/main/java/com/fundation/logic/controller/Controller.java [hunk 7]
```diff
@@ -256,6 +256,11 @@ private Criteria getImageCriteria() {
         if (height.length() == 0) {
             height = "0";
         }
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/controller/Controller.java:259]: Unnecessary line

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 8]
```diff
@@ -25,31 +25,31 @@
...
 
     /**
      * This method read all metadata.
-     *
-     * @return
+     * @return void
      */
     public void readAll() {
         String metadata = null;
         list = new ArrayList<>();
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:44]: define the type for the arrayList content

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 9]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
-        String imageMimeType = mimeType.substring(0,9);
-        if ((imageMimeType.contains("MIME Type"))) {
-            if (mimeType.contains("image")) {
-                searchMimeType = "image";
+        String mimetype2 = mimeType;
+        String audioMimeType = mimeType.substring(0, 4);
+        if ((audioMimeType.contains("MIME"))) {
+            if (mimetype2.contains("image")) {
+                int initIndex = mimetype2.indexOf(":");
+                int freeSpace = 2;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:71]: try to define a constant for freeSpace variable

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java (hunk 1 above)
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:40]: void methods don't have return documentation

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java (hunk 2 above)
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:63]: void methods don't have return documentation

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java (hunk 3 above)
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:82]: void methods don't have return documentation

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 10]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
-    /**
-     * This method returns height image.
-     */
-    public static String getHeight() {
-        return searchHeight;
-    }
-
     /**
      * This method returns metadata image Width.
+     * @return void
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:97]: void methods don't have return documentation

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java (hunk 5 above)
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:113]: void methods don't have return documentation

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 11]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
-     *
-     * @return
+     * @return void
      */
     public static void mimeType(String mimeType) {
-        String imageMimeType = mimeType.substring(0,9);
-        if ((imageMimeType.contains("MIME Type"))) {
-            if (mimeType.contains("image")) {
-                searchMimeType = "image";
+        String mimetype2 = mimeType;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:66]: Define more significant name for mimetype2 variable, for example, auxiliarMimetype or another.

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 12]
```diff
@@ -60,39 +60,41 @@ public void readAll() {
...
 
     /**
      * This method returns metadata image height.
+     * @return void
      */
     public static void height(String height) {
         if ((height.contains("Image Height"))) {
             int initIndex = height.indexOf(":");
             int endIndex = height.length();
             int freeSpace = 2;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:88]: try to define a constant for freeSpace variable

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 13]
```diff
@@ -101,47 +103,74 @@ public static void width(String width) {
...
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:103]: try to define a constant for freeSpace variable

File: src/main/java/com/fundation/logic/common/MetadataImageExtractor.java [hunk 14]
```diff
@@ -101,47 +103,74 @@ public static void width(String width) {
...
      */
-    public static void getColorSpace(String colorSpace) {
-        if ((colorSpace.contains("Color Space"))) {
-            int initIndex = colorSpace.indexOf(":");
-            int endIndex = colorSpace.length();
-            int freeSpace = 2;
-            colorSpace = colorSpace.substring(initIndex + freeSpace, endIndex);
-            searchColorSpace = colorSpace;
+    public static void getColorSpace(String mimeType) {
+        String mimetype2 = mimeType;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataImageExtractor.java:116]: Define more significant name for mimetype2 variable, for example, auxiliarMimetype or another.

File: src/main/java/com/fundation/logic/common/MetadataVideoExtractor.java [hunk 15]
```diff
@@ -66,16 +67,19 @@ public void readAll() {
     }
 
     /**
-     * This method get all metadata and set in each method for get a specific metadata.
+     * This method read mime type.
      * @return void
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataVideoExtractor.java]: void methods don't have return documentation.

File: src/main/java/com/fundation/logic/common/MetadataVideoExtractor.java [hunk 16]
```diff
@@ -66,16 +67,19 @@ public void readAll() {
...
     public static void mimeType(String mimeType) {
-        if ((mimeType.contains("MIME Type"))) {
-            if (mimeType.contains("video")) {
-                searchMimeType = "Video";
+        String mimeTypeVideo = mimeType;
+        String audioMimeType = mimeType.substring(0, 4);
+        if ((audioMimeType.contains("MIME"))) {
+            if (mimeTypeVideo.contains("video")) {
+                int initIndex = mimeTypeVideo.indexOf(":");
+                int freeSpace = 2;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataVideoExtractor.java:77]: try to define a constant for freeSpace variable.

File: src/main/java/com/fundation/logic/controller/Controller.java (hunk 7 above)
Comment (aburgosu) [inline src/main/java/com/fundation/logic/controller/Controller.java:259]: delete white lines

File: src/main/java/com/fundation/logic/controller/Controller.java [hunk 17]
```diff
@@ -256,6 +256,11 @@ private Criteria getImageCriteria() {
...
+        String colorSpace = searchFrame.getSearchTabs().getSplitPanelSearch().getSearchAdvanceTab().getPanelImageAdvanced()
+                .getComboBoxColorSpaceData().getSelectedItem().toString();
+
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/controller/Controller.java:262]: delete white lines

**Pull request closed without merging**